src/tests/test_python.py

```diff
@@ -55,10 +56,20 @@ def test_load_package_after_convert_string(selenium):
     """
     See #93.
     """
-    selenium.run("import sys; x = sys.version")
-    selenium.run_js("let x = pyodide.runPython('x'); console.log(x);")
+
```

Comment:
Maybe we can remove this testcase. I think it is quite old and already covered by other test cases.